### DISCUSSION TITLE
Is aio-libs.discourse.group dead?

### DISCUSSION THREAD

**Discussion by pepoluan** [aio-libs/aiosmtpd#340] [General]
Though I still find references to it if I do a search, trying to access https://aio-libs.discourse.group/ fails, and using down detectors all seem to agree that it's down.

Is it going to be a permanent down or just a temporary one?

**Comment by pepoluan** [comment]
Answer from `aiohttp` maintainers + their recent PR that removes link to the Discourse instance indicates that it's gone for good.

So, likewise, I've removed links to the Discourse instance from our readme files.

**Reply by webknjaz** [reply]
The current recommendation is to use GH Discussions and Matrix for chatting.
